crates/starknet/src/starknet/mod.rs

```diff
@@ -159,6 +175,95 @@ impl Starknet {
 
         this.restart_pending_block()?;
 
+        // Re-execute transactions
+        if transactions.is_some() {
+            // is_some() is need here?
+            for (_hash, transaction) in transactions.unwrap_or_default().iter() {
+                match transaction.inner.clone() {
+                    Transaction::Declare(DeclareTransaction::Version0(_tx)) => {
+                        panic!("DeclareTransactionV0V1 is not supported");
+                    }
+                    Transaction::Declare(DeclareTransaction::Version1(tx)) => {
+                        let contract_class = this
+                            .state
+                            .contract_classes
+                            .get(&tx.class_hash)
+                            .expect("Failed to load Cairo0ContractClass from state");
```

Comment:
Please create corresponding errors in errors.rs module. This code can't use expect/panics

Reply:
Also for all cases below